### PR TITLE
Add public WooPay link in registration copy

### PR DESCRIPTION
**Pull request by hsingyuc**
Fixes 917-gh-Automattic/woopay

#### Changes proposed in this Pull Request
Add public WooPay link in the registration copy

#### Screenshots
*Before*

Screen Shot 2022-08-03 at 4 19 02 PM

*After*

Screen Shot 2022-08-03 at 4 19 53 PM

#### Testing instructions
1. Go to the shop and put a product in the cart
2. Go to checkout
3. Find 'Remember your details?' and tick 'Save my information for faster checkouts'
4. Find '......at other stores powered by WooPay.'
5. See WooPay with the link style and click on the link
6. It should open another page and link to https://woocommerce.com/woopay/

*

-------------------

- [X] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [X] Covered with tests (or have a good reason not to test in description ☝️)
- [X] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: client/components/platform-checkout/save-user/additional-information.js [hunk 1]
```diff
@@ -17,11 +18,24 @@ const AdditionalInformation = () => {
...
 				<span>
-					{ __(
-						'Enter your mobile phone number to save your checkout information for faster checkouts here, ' +
-							'and at other stores powered by WooPay.',
-						'woocommerce-payments'
-					) }
+					{ interpolateComponents( {
+						mixedString: __(
+							'Enter your mobile phone number to save your checkout information for faster checkouts here, ' +
+								'and at other stores powered by {{wooPayLink/}} .',
```
Comment (brianyu28) [inline client/components/platform-checkout/save-user/additional-information.js:24]: Looks like there's an extra space between the end of `{{wooPayLink/}}` and the `.`

**Review by brianyu28 — CHANGES_REQUESTED** [review]
Thanks for working on this, @hsingyuc! One very small change and then this looks good to merge.

File: client/components/platform-checkout/save-user/additional-information.js (hunk 1 above)
Comment (hsingyuc) [reply client/components/platform-checkout/save-user/additional-information.js]: Thank you for the feedback, @brianyu28!

**Review by brianyu28 — APPROVED** [review]
Thanks, @hsingyuc!

**Pull request merged**